src/Orleans/Serialization/OrleansJsonSerializer.cs

```diff
@@ -136,7 +132,34 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
         }
     }
 
-    class GrainIdConverter : JsonConverter
+    public class IPEndPointConverter : JsonConverter
```

Comment:
My thinking was: For public types make public converter so others can reuse it, for internal types make internal converter.

I don't mind changing it, if you think it should be.